packages/commonwealth/server/util/validateTopicThreshold.ts

```diff
@@ -27,20 +27,25 @@ const validateTopicThreshold = async (
             // only support thresholds on token forums
             // TODO: can we support for token-backed DAOs as well?
             type: ChainType.Token,
-          } as WhereOptions<ChainAttributes>
+          } as WhereOptions<ChainAttributes>,
         },
       ]
     });
     if (!topic?.chain) {
       // if associated with an offchain community, or if not token forum, always allow
       return true;
     }
+    const communityContracts = await models.CommunityContract.findOne({
```

Comment:
You should be able to query this directly from the Chain rather than explicitly referencing the through table.

Reply:
gonna leave it now because we're not fetching the ChainInstance, but receiving a ChainAttribute via the Topic's include statement. moreover, this route doesn't matter long-term as it'll get deprecated with the next Rules push.